File: src/main/java/com/optica/dto/CustomerDto.java
package com.optica.dto;

import javax.validation.constraints.Max;
import javax.validation.constraints.Min;
import javax.validation.constraints.NotNull;

import org.hibernate.validator.constraints.Email;

public class CustomerDto {

	@Email
	private String name;
	
	@NotNull
	@Max(value = 18)
	@Min(15)
	private Integer age;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public Integer getAge() {
		return age;
	}

	public void setAge(Integer age) {
		this.age = age;
	}
	
	
	
	

}

File: src/main/kotlin/com/optica/dto/CustomerDto.kt
package com.optica.dto

import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull

import org.hibernate.validator.constraints.Email

class CustomerDto {

    @Email
    var name: String? = null

    @NotNull
    @Max(value = 18)
    @Min(15)
    var age: Int? = null


}
